Dockerfile

```
@@ -25,12 +25,14 @@ COPY --chown=golang:root internal ./internal
 
 RUN go build -v -o joy-generator ./cmd/server
 
-FROM scratch AS prod
+FROM alpine AS prod
```

```suggestion
FROM alpine AS prod
```

Should specify a version since we aren't using scratch anymore, this is implicitly will use `alpine:latest`

---

using `alpine:3.18`